src/Shape/Stack.elm

```diff
@@ -86,7 +86,7 @@ offsetDiverging series =
         [] ->
             []
 
-        first :: rest ->
+        _ :: _ ->
```

Comment:
Actually, do we even need this case statement?

```
---- Elm 0.19.1 ----------------------------------------------------------------
Say :help for help and :exit to exit! More at <https://elm-lang.org/0.19.1/repl>
--------------------------------------------------------------------------------
> import List.Extra as List
> [] |> List.transpose |> List.map (Debug.log "called") |> List.transpose
[] : List (List a)
>
```

Reply:
We do not :), I'll take it out.